renovate.json

```diff
@@ -2,6 +2,11 @@
   "extends": [
     "config:base"
   ],
+  "force": {
+    "constraints": {
+      "node": "= 12.20"
```

Comment:
We might want to document that this version should match the node base image versions in `docker-compose.yml`, so that we don't forget to update it here.

Reply:
Might be nice, but I thought JSON doesn't support comments